app/views/admin/loan_questions/_form.slim

```diff
@@ -16,9 +16,21 @@
 
     = f.input :data_type, as: :select, label: t('loan_questions.data_type'),
       collection: @data_types
-      
+
```

Comment:
Does everyone have auto-truncate turned on in their editor? I'm still seeing whitespace like this showing up...